### PR TITLE
Fix deprecation warnings from webpack 5

### PR DESCRIPTION
**Pull request by ronami**
### Summary

Similar to https://github.com/wix-incubator/tpa-style-webpack-plugin/pull/23, this PR tries to fully support webpack v5. While this plugin works fine with webpack v5, there's a lot of noise from deprecation warnings.

I fixed the deprecation warnings, while still supporting webpack v4. I used a flag to determine what APIs to use. I think it's a lot easier than maintaining two major versions of this plugin.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -14,35 +17,54 @@ class RtlCssPlugin {
       filename: '[name].rtl.css',
     };
   }
+  
+  processAssets(compilation, callback) {
+    // Explore each chunk (build output):
+    compilation.chunks.forEach(chunk => {
+      // Explore each asset filename generated by the chunk:
+      const files = isWebpack5 ? [...chunk.files] : chunk.files;
```
Comment (ranyitz) [inline src/index.js:25]: What is the type of chunk?
Why do you need to spread it into an array?

**Review by ranyitz — APPROVED** [review]
LGTM

File: src/index.js (hunk 1 above)
Comment (ronami) [reply src/index.js:25]: Oh, forgot to explain that with the Github UI. Webpack 5 changed this from an array to a `Set`. It's easier to turn in into an array for filtering/mapping it bellow.

Comment (ranyitz) [reply src/index.js:25]: I assumed, but wanted to be sure :) Thanks for explaining

**Pull request merged**